modules/Bio/EnsEMBL/Compara/RunnableDB/PrepareMaster/UpdateGenomesFromMetadataFactory.pm

```diff
@@ -381,7 +381,7 @@ sub _parse_rr_genome_files {
         next if $_ !~ /[a-z]/;
         # the strain column can be blank, leading to mismatch of column numbers
         $_ = $_ . 'NULL' if $_ !~ /[a-z0-9]$/;
```

Comment:
Now that `map_row_to_header` uses a LIMIT of -1, it will not remove empty trailing columns, so It should be possible to remove these lines.

Reply:
🦺  - thanks